### PR TITLE
Fix fifoset (3x)

### PR DESCRIPTION
**Pull request by shargon**
Before the patch it return null values

### CODE REVIEW

**Comment by shargon** [comment]
We use the known hash for remove task here

image

But before the patch we get this

image

So know hashes was not working as expected

File: neo.UnitTests/UT_FifoSet.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_FifoSet
+    {
+        [TestMethod]
+        public void FifoSetTest()
+        {
+            var a = UInt256.Zero;
+            var b = new UInt256();
```
Comment (igormcoelho) [inline neo.UnitTests/UT_FifoSet.cs]: UInt256.Zero? Better than `new UInt256()`, no?

Comment (shargon) [reply neo.UnitTests/UT_FifoSet.cs]: I want the same data, with different objects

Comment (igormcoelho) [reply neo.UnitTests/UT_FifoSet.cs]: Better to use two `new UInt256()`. Isn't it clearer?

Comment (shargon) [reply neo.UnitTests/UT_FifoSet.cs]: Yes, Is the same

**Comment by vncoelho** [comment]
@shargon and @igormcoelho, please check this additional tests for fifo size.

**Comment by vncoelho** [comment]
@shargon, there is something strangeeee..

Tests of 5547a8e should had not passed!!

**Comment by shargon** [comment]
@vncoelho it passed now

**Comment by vncoelho** [comment]
@shargon, commit https://github.com/neo-project/neo/pull/905/commits/5547a8e2fbdd27b8baea0a25713ad262d1d8246b had errors and passed, something is strange.

**Comment by vncoelho** [comment]
@shargon, I mean, this commit was wrong, it should not have passed.

image

Something weird happen. I will investigate it a lit bit more.

**Comment by igormcoelho** [comment]
Strange, I'll take a lool.

**Comment by shargon** [comment]
The test pass locally, please revert the changes, i will review the problem in few hours

**Comment by vncoelho** [comment]
Shargon, the problem was that all tests was passing locally, even the wrong...ajajajja

**Comment by shargon** [comment]
now pass, what is wrong?

File: neo.UnitTests/UT_FifoSet.cs [hunk 2]
```diff
@@ -27,6 +27,28 @@ public void FifoSetTest()
             Assert.IsTrue(set.Add(c));
 
             CollectionAssert.AreEqual(set.ToArray(), new UInt256[] { a, c });
+            
+                        var d = new UInt256(new byte[32] {
```
Comment (shargon) [inline neo.UnitTests/UT_FifoSet.cs]: indent please

**Pull request merged**